pkg/scheduler/apis/config/validation/validation.go

```diff
@@ -131,37 +189,37 @@ func ValidatePolicy(policy config.Policy) error {
 }
 
 // validateExtenders validates the configured extenders for the Scheduler
-func validateExtenders(fldPath *field.Path, extenders []config.Extender) field.ErrorList {
-	allErrs := field.ErrorList{}
+func validateExtenders(fldPath *field.Path, extenders []config.Extender) []error {
+	var errs []error
 	binders := 0
 	extenderManagedResources := sets.NewString()
 	for i, extender := range extenders {
 		path := fldPath.Index(i)
 		if len(extender.PrioritizeVerb) > 0 && extender.Weight <= 0 {
-			allErrs = append(allErrs, field.Invalid(path.Child("weight"),
+			errs = append(errs, field.Invalid(path.Child("weight"),
 				extender.Weight, "must have a positive weight applied to it"))
 		}
 		if extender.BindVerb != "" {
 			binders++
 		}
 		for j, resource := range extender.ManagedResources {
 			managedResourcesPath := path.Child("managedResources").Index(j)
-			errs := validateExtendedResourceName(v1.ResourceName(resource.Name))
-			for _, err := range errs {
-				allErrs = append(allErrs, field.Invalid(managedResourcesPath.Child("name"),
+			validationErrors := validateExtendedResourceName(v1.ResourceName(resource.Name))
```

Comment:
Change this function to build `field.Error`s, still wrapped as `[]error`. Probably need to pass `path` to the function.

Reply:
I guess on issues can escape your eyes, thumb up! ;)